### ISSUE TITLE
I was hidden by another user.

### ISSUE THREAD

**Issue by DuelingFoodon** [halvnykterist/rating-update#57]
My profile was hidden by another user and was hoping there is a way to reverse this.

**Comment by nemasu** [comment]
I don't think this is possible, are you checking on https://puddle.farm ? It's being hosted there for now.
